packages/cspell/bin.js

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env node
+/* eslint-disable @typescript-eslint/no-var-requires */
 
 'use strict';
 
-const app = require('./dist/app');
+const app = require('./dist/src/app');
```

Comment:
Should not be in `src`.

Reply:
`package.json` is getting copied into `./dist`

Reply:
Yeah, that was a change because of moving the package.json from `require` to `import`. I can undo that if you want

Reply:
Let's undo that part, because it copies the wrong `package.json`.

Reply:
OK, I've undone it. Looks like I did the same in cspell-tools, so I'll do a separate patch to revert that